front/src/weekview/Weekview.module.scss

```diff
@@ -7,21 +9,21 @@
   max-width: 80%;
 }
 
-.big-container {
+.bigContainer {
   width: 100vw;
   max-width: 1025px;
   margin: auto;
 }
 
-.flex-container {
+.flexContainer {
   display: flex;
   flex-wrap: nowrap;
   justify-content: center;
   border: 1px solid;
   border-right: 0px;
 }
 
-.flex-container2 {
+.flexContainer2 {
```

Comment:
Since .flexCointainer shares a lot of the same styles, I'd extend it here and delete the duplicates and override conflicting styles. Something like this should work:
`.flexContainer2 {
  @extend .flexContainer;
  text-align: center;
  border-width: 1px 1px 0 1px;
}`

Reply:
It's not a huge deal so no worries about looking into it